### PR TITLE
Fix LSRainbowMultPlusShield

### PR DESCRIPTION
**Pull request by chesteripz**
An attempt to fix ATK multiplier of 4636, I'm not sure what does `data[5]` mean, but this seems to be okay for existing skills in game

### CODE REVIEW

**Comment by RheingoldRiver** [comment]
Bug report:

image

File: etl/pad/raw/skills/leader_skill_info.py [hunk 1]
```diff
@@ -1496,7 +1496,7 @@ def __init__(self, ms: MonsterSkill):
         self.min_rcv = 1
         self.rcv_step = 0
 
-        max_atk = self.min_atk + self.atk_step * (5 - data[5])
+        max_atk = self.min_atk + self.atk_step * len(binary_con(data[5]))
```
Comment (chasehult) [inline etl/pad/raw/skills/leader_skill_info.py:1499]: Okay, I have NO clue what `data[5]` is either, but this logic doesn't feel right.  I don't think this is a bitflag.  I believe the right logic here is to use `(5 - data[1])`, or more clearly, `(self.max_attr - self.min_attr)`.  I would also make another attribute called `self.unknown5` and set it to `data[5]`.  I really do have NO clue what it does.  It's also of note that making this change will have no effect.  The only monster that uses `data[5]` with a non-zero `data[4]` is Gorkaos, and 3 is the minimum number of attributes.

File: etl/pad/raw/skills/leader_skill_info.py [hunk 2]
```diff
@@ -1487,7 +1487,7 @@ class LSRainbowMultPlusShield(LeaderSkill):
     skill_type = 170
```
Comment (chasehult) [inline etl/pad/raw/skills/leader_skill_info.py:1487]: Unless I'm mistaken, no part of r ragdra's LS uses this skill type.

**Review by chasehult — CHANGES_REQUESTED** [review]

File: etl/pad/raw/skills/leader_skill_info.py (hunk 1 above)
Comment (chasehult) [reply etl/pad/raw/skills/leader_skill_info.py:1499]: Okay.  I think this should be `(self.max_attr - self.min_attr)`

File: etl/pad/raw/skills/leader_skill_info.py [hunk 3]
```diff
@@ -1487,7 +1487,7 @@ class LSRainbowMultPlusShield(LeaderSkill):
...
 
     def __init__(self, ms: MonsterSkill):
-        data = merge_defaults(ms.data, [0, 0, 100, 0, 0, 5])
+        data = merge_defaults(ms.data, [0, 0, 100, 0, 0, 0])
         self.match_attributes = binary_con(data[0])
         self.min_attr = data[1]
         self.max_attr = 5
```
Comment (chasehult) [inline etl/pad/raw/skills/leader_skill_info.py:1493]: This should be `min(5, self.min_attr + data[5])`

File: etl/pad/raw/skills/leader_skill_info.py [hunk 4]
```diff
@@ -1487,7 +1487,7 @@ class LSRainbowMultPlusShield(LeaderSkill):
     skill_type = 170
 
     def __init__(self, ms: MonsterSkill):
-        data = merge_defaults(ms.data, [0, 0, 100, 0, 0, 5])
+        data = merge_defaults(ms.data, [0, 0, 100, 0, 0, 0])
```
Comment (chasehult) [inline etl/pad/raw/skills/leader_skill_info.py:1490]: This should stay `5` I think

**Review by chasehult — CHANGES_REQUESTED** [review]

**Pull request merged**